source/common/config/new_grpc_mux_impl.h

```diff
@@ -117,8 +117,7 @@ class NewGrpcMuxImpl
   // the whole subscription, or if a removed name has no other watch interested in it, then the
   // subscription will enqueue and attempt to send an appropriate discovery request.
   void updateWatch(const std::string& type_url, Watch* watch,
-                   const std::set<std::string>& resources,
-                   const bool creating_namespace_watch = false);
+                   const std::set<std::string>& resources, bool creating_namespace_watch = false);
```

Comment:
Should `creating_namespace_watch` stay const?

Reply:
Nup, there is no semantic meaning in declaring a scalar parameter const at the interface level.